src/systems/ec/implementations/react/packages/radio/src/RadioGroup.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import RadioButton from './RadioButton';
+
+const RadioGroup = ({
+  labelId,
+  items,
+  name,
+  helperText,
+  label,
+  hideLabel,
+  invalid,
+  invalidText,
+  binary,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-radio__group', {
+    'ecl-radio__group--invalid': invalid,
+    'ecl-radio__group--binary': binary,
+  });
+
+  return (
+    <div
```

Comment:
Why not a fieldset?
https://www.w3.org/WAI/tutorials/forms/grouping/

Reply:
I tried it first, but styling of fieldset and legend is a nightmare.
Basically I didn't succeed in doing it

Reply:
ok for the role=radiogroup then